R/importCNA.GISTIC2.R

```diff
@@ -0,0 +1,105 @@
+# importCNA.GISTIC2.R
+#
+#
+#'@function Imports copy number aberration data from Firehose database,
+# produces an rCNA object saved as a RDS file.
+#
+#
+#
+#'@param fFHG Local fully qualified path of Firehose GISTIC2 CNA file.
+#'@param dCNA The local fully qualified path of a directory to store the rCNA RDS file(s).
+#'@param silent Boolean. Default: FALSE. Whether or not to write progress information to console.
+#'@param noLog Boolean. Default: FALSE. Whether or not to log results.
+#'@return gG object
+#'@notRun {
+#'@family importM.COSMIC, importM.TCGA, importM.GISTIC2, importM.ProjectGenie
+#}
+#'@example
+# fFHG <- "inst/extdata/devCNA.txt"
+# dCNA <- "inst/extdata/dCNA"
+# silent <- FALSE
+# writeLog <- TRUE
+# importCNA.GISTIC2("inst/extdata/devCNA.txt", "inst/extdata/dCNA")
```

Comment:
don't need to assign fFHG and dCNA if you aren't using them during importCNA.GISTIC2 call.